### PR TITLE
[DEV-10849] - re-order commands

### PR DESCRIPTION
**Pull request by tomerstein**

### CODE REVIEW

File: pkg/commands/access/actions/reset.go [hunk 1]
```diff
@@ -16,10 +16,9 @@ const (
 
 func AccessReset() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "reset-credentials [id]",
-		GroupID: Group.ID,
-		Short:   "Reset access session credentials",
-		Args:    cobra.ExactArgs(1), // This will enforce that exactly 1 argument must be provided
+		Use:   "reset-credentials [id]",
```
Comment (naorgu) [inline pkg/commands/access/actions/reset.go:19]: in `use.go` you put <> instead of []
let's align

File: pkg/commands/auth/actions/auth.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import (
+	"github.com/spf13/cobra"
+)
+
+func Profiles() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "profiles",
+		Short:   "Manage the CLI's profiles",
+		Aliases: []string{"profile"},
+		GroupID: Group.ID,
```
Comment (naorgu) [inline pkg/commands/auth/actions/auth.go:12]: let's rename
```suggestion
		GroupID: AuthenticationCommandsGroup.ID,
```

File: pkg/groups/managment.go [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package groups
+
+import "github.com/spf13/cobra"
+
+var ManagementCommandsGroup = &cobra.Group{
```
Comment (naorgu) [inline pkg/groups/managment.go:5]: I wonder if we can think of a better name...

File: pkg/groups/managment.go [hunk 4]
```diff
@@ -0,0 +1,8 @@
+package groups
```
Comment (naorgu) [inline pkg/groups/managment.go:1]: maybe better to move other groups to this module ?

**Review by naorgu — APPROVED** [review]

File: pkg/groups/managment.go (hunk 4 above)
Comment (tomerstein) [reply pkg/groups/managment.go:1]: you are right, I will move them

File: pkg/groups/managment.go (hunk 3 above)
Comment (tomerstein) [reply pkg/groups/managment.go:5]: maybe "Core commands"?
I saw the "Management" one in another CLI tool

**Pull request merged**